src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+//spread 
+//TimeStamp has time prop
+const ChatEntry = ({ updateChat, ...message}) => {
```

Comment:
Nice use of destructuring with rest syntax to get the callback function separate from the rest of the message data.